TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8317,11 +8318,28 @@ sqlGotoHostLabel: (SQL_GOTO | SQL_GO SQL_TO) ColonSeparator? hostLabel=UserDefin
 hostVariable: ColonSeparator mainVariable=UserDefinedWord ((indicator)? ColonSeparator indicatorVariable=UserDefinedWord)?;
 indicator: ({string.Equals(CurrentToken.Text, "INDICATOR", System.StringComparison.OrdinalIgnoreCase) }? KeywordINDICATOR=UserDefinedWord);
 
+sqlConstant: SQL_DecimalFloatingPointLiteral | SQL_BinaryStringLiteral | SQL_GraphicStringLiteral | AlphanumericLiteral | HexadecimalAlphanumericLiteral | IntegerLiteral | DecimalLiteral | FloatingPointLiteral | datetime_constant | SQL_NULL;
```

Comment:
sqlConstant is not used in `alterSequenceStatement`. It was missing from a previous PR ?

Reply:
Cache size is an integer-constant  .

Reply:
@smedilol is right, since the 'sqlConstant' ANTLR rule is not used, it should be removed from this PR. However the builder will create `SqlConstant` instances for the different numeric literals used here.